benchmark/cluster/vgg16/vgg16_fluid.py

```diff
@@ -11,7 +11,25 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""VGG16 benchmark in Fluid"""
+"""VGG16 benchmark in Fluid
```

Comment:
Is it better to change these comments to a separate bash script file?

Reply:
Done